src/main/scala/io/iohk/ethereum/blockchain/sync/regular/RegularSync.scala

```diff
@@ -92,7 +110,22 @@ object RegularSync {
       )
     )
 
-  sealed trait RegularSyncMsg
-  case object Start extends RegularSyncMsg
-  case class MinedBlock(block: Block) extends RegularSyncMsg
+  case class ProgressState(startedFetching: Boolean, initialBlock: BigInt, currentBlock: BigInt, targetBlock: BigInt) {
```

Comment:
I am little reluctant to name `targetBlock` in `RegularSync` as it suggests there is some kinda target which is not true ? Maybe `bestKnownNetworkTop` or something simillar ?